sdks/go/pkg/beam/core/runtime/exec/fn.go

```diff
@@ -99,6 +124,11 @@ func newInvoker(fn *funcx.Fn) *invoker {
 	if n.outErrIdx, ok = fn.Error(); !ok {
 		n.outErrIdx = -1
 	}
+	// TODO(@damccorm) - add this back in once BundleFinalization is implemented
```

Comment:
Prefer to annotate TODOs with JIRA tickets instead of people. Most of the legacy TODOs use people because they were added before the SDK was externalized.

So here, TODO(BEAM-10976)

Reply:
Done - that makes sense, and I'm much happier with that pattern 😃